test/gtest/uct/test_md.cc

```diff
@@ -558,6 +558,30 @@ UCS_TEST_P(test_md, mem_query) {
     }
 }
 
+UCS_TEST_P(test_md, tl_resource_desc_v1_v2) {
+    uct_md_query_tl_resources_params_t params = {};
+    uct_tl_resource_desc_t *v1;
+    uct_tl_resource_desc_v2_t *v2;
+    unsigned num_v1, num_v2;
+
+    ASSERT_UCS_OK(uct_md_query_tl_resources(md(), &v1, &num_v1));
+    ASSERT_UCS_OK(uct_md_query_tl_resources_v2(md(), &v2, &num_v2, &params));
+    ASSERT_TRUE(num_v2 > 0);
+    ASSERT_EQ(num_v2, num_v1);
```

Comment:
maybe EXPECT_EQ, to avoid resource leak on failure

Reply:
fixed